org-clubhouse.el

```diff
@@ -175,21 +175,21 @@ If unset all projects will be synchronized")
        (rx "[[" (one-or-more anything) "]"
            "[" (group (one-or-more digit)) "]]")
        clubhouse-id-link)
-      (string-to-int (match-string 1 clubhouse-id-link)))
+      (string-to-number (match-string 1 clubhouse-id-link)))
      ((string-match-p
        (rx buffer-start
            (one-or-more digit)
            buffer-end)
        clubhouse-id-link)
-      (string-to-int clubhouse-id-link)))))
+      (string-to-number clubhouse-id-link)))))
```

Comment:
https://github.com/emacs-mirror/emacs/blob/master/etc/NEWS.26#L1332-L1333)

https://www.gnu.org/software/emacs/manual/html_node/elisp/String-Conversion.html

Reply:
yep, I had noticed this as well